.travis.yml

```diff
@@ -19,8 +19,9 @@ install:
   - pip install coverage
 
 script:
-  - flake8 anybox 
-  - nosetests anybox --with-coverage --cover-tests --with-doctest --cover-package anybox 
+  - flake8 anybox
+  - nosetests anybox --with-coverage --cover-tests --with-doctest --cover-package anybox
```

Comment:
`flake8 anybox` doesn't cover as much as `setup.py flake8`

```
$ flake8 anybox
# Nothing
$ python setup.py flake8
running flake8
setup.py:20:14: E225 missing whitespace around operator
setup.py:20:22: E231 missing whitespace after ','
setup.py:53:7: E121 continuation line under-indented for hanging indent
```

We could change the doc